ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/restore/view/RestoreView.kt

```diff
@@ -159,88 +184,69 @@ private fun RestoreTopAppBar(onBack: () -> Unit, onClear: () -> Unit) {
     )
 }
 
-@Suppress("UNUSED_PARAMETER")
+// TODO [#672] https://github.com/zcash/secant-android-wallet/issues/672
```

Comment:
A really minor thing - we agreed to use a bit more detailed two-lined TODO comment with also a description of the linked issue, so we don't have to open the link. E.g. like this [one](https://github.com/zcash/secant-android-wallet/blob/381af575efbe87b9d62c7a410fadab4ded1b1a1a/gradle.properties#L146).

Reply:
> * Why do we now use two titles here? I mean the one _Create a wallet_ in toolbar and the second bold one _Wallet import_.

 i added the second title to reflect wireframes, but i agree that one is probably enough...

> * I'd like to distinguish visually already confirmed chips from those in the suggestion box (e.g. with a slightly different background color). What do you think?

 i think this is a good idea!

> * I was able to reproduce a UI state in which a user can't see what he/she typing. We can e.g. make the whole bottom part (above keyboard) scrollable so it's not deformed then?

i think that if we lock screen orientation to Portrait it will resolve the issue, I'm not sure if we need to support landscape orientation

@geffenz  any thoughts on this?

Reply:
Locking to portrait hides significant Android lifecycle bugs during development and therefore we should continue suffering this pain of screen rotation for now.

Reply:
Yes, agree. We should be prepared for landscape mode from the beginning. Once the MVP is out and things stabilized, we can move towards having a separate Composes for the landscape orientation. Until then, we're good with these mixed orientation screens.

Reply:
I absolutely agree on both suffering the pains now and later creating different Composes per orientation!